doc/ref_model/chapters/chapter05.rst

```diff
@@ -630,10 +630,9 @@ Compute Resources
 | infra.hw.cpu.cfg.003 | NUMA alignment                  | NUMA alignment enabled and BIOS | N           | Y           |
 |                      |                                 | configured to enable NUMA       |             |             |
 +----------------------+---------------------------------+---------------------------------+-------------+-------------+
-| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | Y                               | Optional    |             |
-|                      | (SMT) SMT enabled that allows   |                                 |             |             |
-|                      | each core to work multiple      |                                 |             |             |
-|                      | streams of data simultaneously  |                                 |             |             |
+| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | SMT enabled that allows each    | Y           |Optional     |
```

Comment:
SMT, even for High Performance (HP), is usually "Y" -- there are very few HP exceptions where you have the second thread disabled (SMT disabled).

High Performance does require that both threads are allocated to the same workload compute resource, not different workloads.